code/modules/mob/living/carbon/human/say.dm

```diff
@@ -143,8 +144,9 @@
 				verb = "yells loudly"
 
 	if((COMIC in mutations) || (locate(/obj/item/organ/internal/cyberimp/brain/clown_voice) in internal_organs))
-		message = "<span class='sans'>[message]</span>"
+		span = "sans"
 
+	message = "<span class='[span]'>[message]</span>"

```

Comment:
... and this should be behind an `if(span)` conditional. Saycode already applies a span class, there's no reason to add an extra wrap for humans.